omero/sysadmins/dropbox.txt

```diff
@@ -38,6 +39,15 @@ the following more specific requirements:
     operating system. Watching a network-attached share (NAS) is strictly
     ***not*** supported.
 
+.. note:: It is likely, but not guaranteed, that DropBox will run on later
+    versions of Windows, however only tested versions are supported by default.
+    Setting the Windows system environment variable
+    :envvar:`OMERO_PLATFORM_CHECK` to ``false`` and restarting the server will
+    allow DropBox to be started on an untested version of Windows.
+
+    If you do successfully run DropBox on an as yet untested version of Windows
+    please do let us know.
```

Comment:
Would probably be nice to add 'via the :community_plone:`mailing lists or forum  <>`. ' just for completeness.